build.sbt

```diff
@@ -197,11 +206,7 @@ lazy val chillJava = module("java").settings(
 lazy val chillStorm = module("storm").settings(
   crossPaths := false,
   autoScalaLibrary := false,
-  resolvers ++= Seq(
-    "Clojars Repository" at "http://clojars.org/repo",
-    "Conjars Repository" at "http://conjars.org/repo"
-  ),
-  libraryDependencies += "storm" % "storm" % "0.9.0-wip9" % "provided"
+  libraryDependencies += "org.apache.storm" % "storm-core" % "1.0.2" % "provided"
```

Comment:
I don't think we can do this. Storm 1.0 actually transitively depends on chill 0.8 so this does not make sense.